ontology/yaml/resources/LIGHTING/entity_types/ABSTRACT.yaml

```diff
@@ -30,6 +30,18 @@ DC:
   uses:
   - brightness_percentage_command
 
+LC_DUAL:
```

Comment:
while technically allowable, I suggest not creating an abstract type name like this to prevent confusion between abstract and canonical type names (this looks like a canonical type name)

I suggest naming this something like `AWIC` (Aisle window illuminance control)

Reply:
Understood. AWIC sounds reasonable.